samples/export-requirements.txt

```
@@ -2,13 +2,14 @@
 --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/pre-release
 --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
 openvino-tokenizers[transformers]~=2025.3.0.0.dev
-optimum-intel[nncf] @ git+https://github.com/huggingface/optimum-intel.git@main
+# optimum-intel main will get issue with VLM gemma3 when transformers < 4.52: https://github.com/huggingface/optimum-intel/issues/1405
+optimum-intel[nncf] @ git+https://github.com/huggingface/optimum-intel.git@d93fd59aebd24ac887deb1eaab7ff6af41b09946
```

We may need to limit transformer version only until optimum-intel fixed compatibility issue 
```suggestion
optimum-intel[nncf] @ git+https://github.com/huggingface/optimum-intel.git@main
```

---

https://github.com/huggingface/optimum-intel/pull/1408 merged, could you please remove pip of optimum intel version

---

yes, remove now, thx